### DISCUSSION TITLE
how to monitor a public url within prometheus operator

### DISCUSSION THREAD

**Discussion by zffocussss** [prometheus-operator/prometheus-operator#4875] [Q&A]
hi guys:
    Provided that I have some metrics which are exposed by public url https://xxx/metrics,how can I scrape and show it on prometheus?

**Comment by simonpasquier** [comment]
> hi guys:

not all contributors are "guys", "folks" would be more appropriate ;)

You probably want to look at the [Probe](https://prometheus-operator.dev/docs/operator/api/#probe) CRD with [ProbeTargetStaticConfig](https://prometheus-operator.dev/docs/operator/api/#probetargetstaticconfig).

**Reply by zffocussss** [reply]
I will look into it.
